src/components/Infopages/TrademarkGuidelines.astro

```diff
@@ -0,0 +1,269 @@
+  >
+    <g clip-path="url(#clip0_17_60)">
+      <g filter="url(#filter0_f_17_60)">
+        <path d="M128.6 0H0V322.2L332.5 211.5L128.6 0Z" fill="#4D07E3"></path>
+        <path d="M0 322.2V400H240H320L332.5 211.5L0 322.2Z" fill="#4C00FF"
+        ></path>
+        <path d="M320 400H400V78.75L332.5 211.5L320 400Z" fill="#B5BFF1"></path>
+        <path d="M400 0H128.6L332.5 211.5L400 78.75V0Z" fill="#7fcef3"></path>
+      </g>
+    </g>
+    <defs>
+      <filter
+        id="filter0_f_17_60"
+        x="-159.933"
+        y="-159.933"
+        width="719.867"
+        height="719.867"
+        filterUnits="userSpaceOnUse"
+        color-interpolation-filters="sRGB"
+      >
+        <feFlood flood-opacity="0" result="BackgroundImageFix"></feFlood>
+        <feBlend
+          mode="normal"
+          in="SourceGraphic"
+          in2="BackgroundImageFix"
+          result="shape"></feBlend>
+        <feGaussianBlur
+          stdDeviation="79.9667"
+          result="effect1_foregroundBlur_17_60"></feGaussianBlur>
+      </filter>
+    </defs>
+  </svg>
+  <div class="max-w-7xl px-8 md:px-12 relative lg:px-32 mx-auto py-32 lg:pt-56">
+    <div class="max-w-3xl">
+      <span
+        class="text-transparent bg-clip-text font-medium tracking-widest bg-gradient-to-r from-cyan-400 via-cyan-500 to-indigo-500 text-xs uppercase"
+        >LAST UPDATED JULY 25, 2024</span
+      >
+      <p class="text-3xl mt-8 font-normal text-white">Trademark guidelines</p>
+    </div>
+  </div>
+</section>
+<section class="relative">
+  <div class="max-w-7xl px-8 md:px-12 lg:px-32 mx-auto pb-12 md:py-12 relative">
+    <div class="grid grid-cols-1 gap-10 xl:gap-24">
+      <div
+        class="prose prose-strong:text-white prose-blockquote:border-l-white prose-a:text-cyan-500 hover:prose-a:text-white prose-headings:font-normal prose-headings:text-white prose-li:marker:text-cyan-500 text-slate-400 prose-code:text-white prose-blockquote:text-white/70 max-w-max"
+      >
+        <p>
+          Below is our trademark policy inspired by <a href="https://dagger.io/trademark-guidelines">Dagger's</a>. We want to make our policy easily
+          accessible, so we have created some FAQs to help steer you in the
+          right direction.
+        </p>
+        <p>Please take a moment to read the policy in its entirety.</p>
+        <p>
+          If you have any questions or doubts about your project using our
+          trademark, please contact us at legal@tuist.io. We are happy to help
+          provide more guidance for your use case.
+        </p>
+        <h2>Trademark FAQs</h2>
+        <h4>
+          I have a repo on GitHub that is related to Tuist. Can I have Tuist in
+          the name?
+        </h4>
+        <p>
+          Yes, as follows. To avoid confusion in the community, your repo must
+          clearly show that it is not directly related to Tuist. We ask that you
+          use the following naming convention: "[YourProjectName]-for-Tuist".
```

Comment:
Not really a thing right now, but once we make it easier to create plugins, I think `tuist-plugin-xxx` should also be a good name choice that makes it obvious this is not an official tuist project